README.md

```diff
@@ -48,22 +47,32 @@ var TryStardust = React.createClass({
 module.exports = TryStardust;
 ```
 
-## Develop
+## Installation
+Start by cloning this repo and installing dependencies:
+
+    git clone git@github.com:TechnologyAdvice/stardust.git stardust && cd $_
+    npm install               # or simply npm i
+
+## Usage
+Once dependencies are installed, start the application with:
+
+    npm start
```

Comment:
We have an install and usage section.  I mean, there are now 2 **Install** and **Usage** sections with conflicting information.  We should specify which one is for which.  I think "using" Stardust ought to mean, developing React applications with it.  In which case, you should `npm i` and add the babel compiler.

### Install and Use Stardust in your project:

![image](https://cloud.githubusercontent.com/assets/5067638/11959834/6019ae32-a884-11e5-8109-5b8260e791d0.png)

### Install and Use (Clone and Develop) Stardust locally:

![image](https://cloud.githubusercontent.com/assets/5067638/11959862/854aeb30-a884-11e5-8561-935586bf2ece.png)